dayliopy/clean_csv.py

```diff
@@ -27,7 +28,14 @@ def clean_row(row, default_activities):
     date = row["full_date"]
     time = row["time"]
     datetime_str = f"{date} {time}"
-    datetime_obj = datetime.datetime.strptime(datetime_str, r"%Y-%m-%d %I:%M %p")
+
+    # Detect if time is 24H or 12H time.
+    datetime_format = ""
+    if 'm' in datetime_str:
```

Comment:
This is more expressive.
```suggestion
    if time.endswith("pm") or time.endswith("pm"):
```